airbyte-integrations/connectors/source-bamboo-hr/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 90916976-a132-4ce9-8bce-82a03dd58788
-  dockerImageTag: 0.3.6
+  dockerImageTag: 0.4.0
```

Comment:
Still miss the update in the `pyproject.toml` 


```
[tool.poetry]
version = "0.4.0"
name = "source-bamboo-hr"
```

@KimPlv is it possible to allow maintainers to make changes?

```
 git push                                                                                                                                                                                             ─╯
ERROR: Permission to sendinblue/airbyte.git denied to marcosmarxm.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
Hi @marcosmarxm i give you some access, tell me if it's ok for you

Reply:
I don't why but still don't have permission. Can you do the changes for me?